templates/pages/checkout.html

```diff
@@ -1 +1,35 @@
-brand.html
+<!doctype html>
+<html>
+<head>
+    <base href="/">
+    <meta charset="UTF-8"/>
+    <meta name="keywords" content="checkout"/>
+    <meta name="description" content="Bigcommerce checkout"/>
+    <meta name="robots" content="noindex, nofollow">
+    <meta name="viewport" content="width=device-width">
+    <link rel="SHORTCUT ICON" href="{{ FaviconUrl }}" />
+    {{{ open_sans_stylesheet }}}
+    {{{ ng_checkout_stylesheet }}}
+    {{{ stylesheet '/assets/css/optimized-checkout.css' }}}
+    {{ getFontsCollection }}
+    {{ OptimizedCheckoutWebdavCss }}
+</head>
+    <body>
+<header class="checkoutHeader optimizedCheckout-header">
+    <div class="checkoutHeader-content">
+        <h1 class="is-srOnly" translate>Checkout</h1>
```

Comment:
We use `lang` helper to load language strings in Stencil. i.e.:

```html
<h1 class="is-srOnly">{{lang 'checkout.title'}}</h1>
```

And we define `checkout.title` in `lang/en.json`